packages/npm/src/version.ts

```diff
@@ -21,11 +21,16 @@ import { IPluginConfig } from "./config";
 import * as utils from "./utils";
 
 export default async function (context: IContext, _config: IPluginConfig): Promise<void> {
+    if (context.workspaces != null) {
+        context.logger.warn("Cannot run npm version in workspaces");
+        return;
+    }
```

Comment:
Curious why that is?
Seems like npm supports running the version command on workspaces since v7 (._.?)
https://docs.npmjs.com/cli/v7/commands/npm-version#workspace

I'm guessing this is a design decision which may explain why we might be requiring a lerna.json ?